src/v2/styles/Message/Message-layout.scss

```diff
@@ -1,5 +1,10 @@
 @use '../utils';
 
+.str-chat {
```

Comment:
@szuperaz do you find this consistent with other layout files?

Reply:
We usually use `calc(var(--str-chat__spacing-px) * 26)` for any kind of sizing, but other than that it looks good to me